### PR TITLE
TimescaleDB 2.4.1 Release notes

### PR DESCRIPTION
**Pull request by NunoFilipeSantos**
# Description

[Short summary of why you created this PR]

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: timescaledb/overview/release-notes/index.md [hunk 1]
```diff
@@ -78,6 +80,19 @@ accidentally triggering the load of a previous DB version.**
...
+**Bug fixes**
+* #3430 Fix havingqual processing for continuous aggregates
+* #3468 Disable tests by default if tools are not found
+* #3468 Fix crash while tracking alter table commands
+* #3494 Improve error message when adding data nodes
+* #3498 Fix continuous agg bgw job failure for PG 12.8 and 13.4
+
+**Thanks**
+* @brianbenns for reporting a segfault with continuous aggregates
+This release adds new experimental features since the 2.3.1 release.
```
Comment (mkindahl) [inline timescaledb/overview/release-notes/index.md:94]: This line should be removed.

**Review by mkindahl — APPROVED** [review]

**Review by mfreed — APPROVED** [review]

File: timescaledb/overview/release-notes/index.md (hunk 1 above)
Comment (NunoFilipeSantos) [reply timescaledb/overview/release-notes/index.md:94]:
```suggestion
```

File: timescaledb/overview/release-notes/index.md [hunk 2]
```diff
@@ -78,6 +80,19 @@ accidentally triggering the load of a previous DB version.**
...
 ## Unreleased
 
+## 2.4.1 (2021-08-19)
+
+**Bug fixes**
+* #3430 Fix havingqual processing for continuous aggregates
+* #3468 Disable tests by default if tools are not found
+* #3468 Fix crash while tracking alter table commands
+* #3494 Improve error message when adding data nodes
+* #3498 Fix continuous agg bgw job failure for PG 12.8 and 13.4
```
Comment (NunoFilipeSantos) [inline timescaledb/overview/release-notes/index.md:90]:
```suggestion
* #3489 Fix continuous agg bgw job failure for PG 12.8 and 13.4
```

File: timescaledb/overview/release-notes/index.md [hunk 3]
```diff
@@ -78,6 +80,18 @@ accidentally triggering the load of a previous DB version.**
...
```
Comment (NunoFilipeSantos) [inline timescaledb/overview/release-notes/index.md]:
```suggestion
* #3489 Fix continuous agg bgw job failure for PG 12.8 and 13.4
```

File: timescaledb/overview/release-notes/index.md [hunk 4]
```diff
@@ -78,6 +80,18 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.4.1 (2021-08-19)
+
+**Bug fixes**
+* #3430 Fix havingqual processing for continuous aggregates
+* #3468 Disable tests by default if tools are not found
+* #3468 Fix crash while tracking alter table commands
```
Comment (NunoFilipeSantos) [inline timescaledb/overview/release-notes/index.md:88]:
```suggestion
* #3462 Fix crash while tracking alter table commands
```

**Pull request merged**